benchmarks/dynamo/pr_time_benchmarks/benchmark_base.py

```diff
@@ -65,6 +67,22 @@ class BenchmarkBase(ABC):
     # number of iterations used to run when collecting instruction_count or compile_time_instruction_count.
     _num_iterations = 5
 
+    def __init__(
+        self,
+        model_type: str,
+        device: str,
+        backend: str = "",
+        mode: str = "",
+        dynamic=False,
```

Comment:
can we default it to None? since if user does not pass we do not know its not dynamic.

Reply:
Sure, let's default this and `fullgraph` to None